test/es-module/test-esm-loader-mock.mjs

```diff
@@ -0,0 +1,39 @@
+// Flags: --loader ./test/fixtures/es-module-loaders/mock-loader.mjs
+import '../common/index.mjs';
+import assert from 'assert/strict';
+import mock from 'node:mock';
```

Comment:
I assume this is created by `mock-loader.mjs`? A comment saying so might be nice.

Reply:
sounds good to add a comment